packages/typegen/src/generate/interfaceRegistry.ts

```diff
@@ -19,7 +19,8 @@ export function generateInterfaceTypes (importDefinitions: { [importPath: string
     Object.entries(importDefinitions).reduce((acc, def) => Object.assign(acc, def), {} as object);
 
     const imports = createImports(importDefinitions);
-    const definitions = imports.definitions;
+    const definitions = { ...imports.definitions, polymesh: { rpc: {}, types: { Weight: 'u32' } } } as object;
```

Comment:
The `polymesh` seems to have snuck in here.

Reply:
Yeah, was using it to test the duplicate type thing